### PR TITLE
feat: Add asc-sample option and include in no-secty-with-ui built compose files

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/CONTRIBUTING.md.

## What is the current behavior?
asc-sample is not an option 

## What is the new behavior?
asc-sample is now an option and is included in the non-secure with UI compose files  so the UI can use it to allow users to manipulate the configurable pipeline.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: compose-builder/Makefile [hunk 1]
```diff
@@ -458,8 +478,8 @@ build:
 	make compose ds-rest ds-virtual arm64
 	make compose ds-rest ds-virtual no-secty
 	make compose ds-rest ds-virtual no-secty arm64
-	make compose ds-rest ds-virtual no-secty ui
-	make compose ds-rest ds-virtual no-secty arm64 ui
+	make compose ds-rest ds-virtual asc-sample no-secty ui
+	make compose ds-rest ds-virtual asc-sample no-secty arm64 ui
```
Comment (jim-wang-intel) [inline compose-builder/Makefile:482]: so this `asc-sample` is only for no-secty and ui?  there is no asc-sample in lines above like 478-480.

File: compose-builder/Makefile [hunk 2]
```diff
@@ -537,6 +557,8 @@ down:
 		-f add-device-virtual.yml \
 		-f add-asc-http-export.yml \
 		-f add-asc-mqtt-export.yml \
+		-f add-asc-sample.yml \
+		-f add-app-rfid-llrp-inventory.yml \
```
Comment (jim-wang-intel) [inline compose-builder/Makefile:561]: why add `rfid-llrp-inventory` here? is this also fixing a missing bug?

File: compose-builder/Makefile (hunk 1 above)
Comment (lenny-intel) [reply compose-builder/Makefile:482]: Yep, this is so the UI has it to allow users to modify the configurable function pipeline. Not needed in any of the other versions.

File: compose-builder/Makefile (hunk 2 above)
Comment (lenny-intel) [reply compose-builder/Makefile:561]: yep, it will missing so add it here.

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**